picard/ui/itemviews.py

```diff
@@ -519,77 +598,27 @@ def add_actions(*args):
-
-            if len(self.selectedItems()) == 1 and obj.release_group:
-                def _add_other_versions():
-                    releases_menu.removeAction(loading)
-                    releases_menu.removeAction(action_more)
-                    heading = releases_menu.addAction(obj.release_group.version_headings)
-                    heading.setDisabled(True)
-                    font = heading.font()
-                    font.setBold(True)
-                    heading.setFont(font)
-
-                    versions = obj.release_group.versions
-
-                    album_tracks_count = obj.get_num_total_files() or len(obj.tracks)
-                    preferred_countries = set(config.setting['preferred_release_countries'])
-                    preferred_formats = set(config.setting['preferred_release_formats'])
-                    ORDER_BEFORE, ORDER_AFTER = 0, 1
-
-                    alternatives = []
-                    for version in versions:
-                        trackmatch = countrymatch = formatmatch = ORDER_BEFORE
-                        if version['totaltracks'] != album_tracks_count:
-                            trackmatch = ORDER_AFTER
-                        if preferred_countries:
-                            countries = set(version['countries'])
-                            if not countries or not countries.intersection(preferred_countries):
-                                countrymatch = ORDER_AFTER
-                        if preferred_formats:
-                            formats = set(version['formats'])
-                            if not formats or not formats.intersection(preferred_formats):
-                                formatmatch = ORDER_AFTER
-                        group = (trackmatch, countrymatch, formatmatch)
-                        # order by group, name, and id on push
-                        heappush(alternatives, (group, version['name'], version['id'], version['extra']))
-
-                    prev_group = None
-                    while alternatives:
-                        group, action_text, release_id, extra = heappop(alternatives)
-                        if group != prev_group:
-                            if prev_group is not None:
-                                releases_menu.addSeparator()
-                            prev_group = group
-                        action = releases_menu.addAction(action_text)
-                        action.setCheckable(True)
-                        if extra:
-                            action.setToolTip(extra)
-                        if obj.id == release_id:
-                            action.setChecked(True)
-                        action.triggered.connect(partial(obj.switch_release_version, release_id))
-
-                    versions_count = len(versions)
-                    if versions_count > 1:
-                        releases_menu.setTitle(_("&Other versions (%d)") % versions_count)
```

Comment:
Yes; that will be T from above comment, we definitively need to get this number since it will help to provide correct info to the user.

It's not possible to not limit display (but we can make the limit configurable.
It's not safe to not limit the number of results (but this is already configurable, though it is only 25 50 75 100, I think it could be anything > 50).
The value displayed in the menu should be the total number of possible releases, and we should adapt the actions to select any of those.
In current code, that's not possible, because `query_limit` is always applied, including in More Details dialog, that's an issue imho.

More Details dialog should use pagination.